com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassForward.hlsl

```diff
@@ -184,8 +184,8 @@ void Frag(PackedVaryingsToPS packedInput,
         // TEMP!
         // For now, the final blit in the backbuffer performs an sRGB write
         // So in the meantime we apply the inverse transform to linear data to compensate.
-        if (!needLinearToSRGB)
-            result = SRGBToLinear(max(0, result));
+        if (needLinearToSRGB && _DebugAllowsRGBConversion != 0)
+            result = LinearToSRGB(max(0, result));
```

Comment:
is there any motivation to switch this one from SRGBToLinear to LinearToSRGB and not the others?

Reply:
This change should be gone after merging 10.x.x/release